### PR TITLE
Sample template to graph stock prices.

### PR DESCRIPTION
**Pull request by vzlatkin**
Added sample template to graph stock prices.  Described here: https://community.hortonworks.com/content/kbentry/8422/visualize-near-real-time-stock-price-changes-using.html

### CODE REVIEW

**Comment by JPercivall** [comment]
Thanks for the PR @vzlatkin!

From a pure readability point of view, I found the flow to be a bit hard to follow. Particularly that the flow zig-zagged and the queue boxes covered up the arrows indicating the flow direction. What do you think of this refactored view? https://gist.github.com/JPercivall/464e887dbca8c7e25cc49e06410fd010

**Comment by JPercivall** [comment]
Couple things to update in the excel, "EvaluateJsonPath" is used in the template but "EvaluateRegularExpression" is marked as being used in the excel instead. Also "ReplaceText" is used in the template but not marked in the excel.

**Comment by vzlatkin** [comment]
done
